ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/clientlibs/site/js/numberinputwidget.js

```diff
@@ -78,8 +78,7 @@
                  this.#regex = new RegExp(localeStr, "g");
                  this.#engRegex = new RegExp(engStr, "g");
              }
-             //initialise the model by the existing value entered in the widget (in case data was entered before initialisation)
-             this.#model.value = this.getValue(this.#widget.value);
+
```

Comment:
Initially, what prompted the addition of this? Comments suggest that its inclusion was intentional.

Reply:
Yes it was intentional since it's possible for someone to enter a value in form field before we initialize the runtime. To avoid data loss this initialization is required

Reply:
I have checked the use-case of the comment mentioned, it is not breaking. don't know why it was added. @nit23uec, could you explain why it was needed?

Reply:
@im-shiv Could you write a test case for the use-case mentioned by nitin

Reply:
@rismehta added